wgpu/src/window/compositor.rs

```diff
@@ -27,7 +27,7 @@ impl Compositor {
         let adapter = instance
             .request_adapter(&wgpu::RequestAdapterOptions {
                 power_preference: if settings.antialiasing.is_none() {
-                    wgpu::PowerPreference::Default
+                    wgpu::PowerPreference::default()
```

Comment:
`wgpu::PowerPreference::Default` no longer exists, maybe there is another way to replicate its behavior?
for now `wgpu::PowerPreference::default()` is used (it always returns `LowPower`)